<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>693</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>693</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cat and Distracted Mouse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="startUpScreen" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>920</width>
      <height>629</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>621</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="startupVecticalLayout">
      <item>
       <widget class="QLabel" name="gameTitle">
        <property name="font">
         <font>
          <family>Purisa</family>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cat and Distracted Mouse</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameDescription">
        <property name="font">
         <font>
          <family>Droid Sans Fallback</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>The purpose of this game is to inform people about the dangers of distracted driving. You play as a field mouse who must dodge all of the vicious cats trying to eat you. Your rat-phone will try to distract you; stay focused, your life depends on it!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gameplayInstructions">
        <property name="font">
         <font>
          <family>Droid Sans Fallback</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dodge the incoming cats by pressing the left or right arrow key. If you get hit by a cat, it is game over!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="startupUpperSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="startupHorizontalLayout">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="startButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Start!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="startupLowerSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="authors">
        <property name="text">
         <string>Created by: August O'Rourke, Luke Hamling, Ryan Dalrymple, and Vincentio Dane</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="mainScreen" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>920</width>
      <height>629</height>
     </rect>
    </property>
    <widget class="World" name="gameplayArea" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>691</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>215</width>
       <height>621</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="statsLayout">
      <item>
       <spacer name="topStatSpacer_MainScreen">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="levelLayout">
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="levelLabel">
          <property name="text">
           <string>Current Level:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="levelValue">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="timeAliveLayout">
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="timeAliveLabel">
          <property name="text">
           <string>Time Alive:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="timeAliveValue">
          <property name="text">
           <string>00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="catsDodgedLayout">
        <item>
         <widget class="QLabel" name="catsDodgedLabel">
          <property name="text">
           <string>Cats Dodged:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="catsDodgedValue">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="notificationsLayout">
        <item>
         <widget class="QLabel" name="notificationsAcknowledgedLabel">
          <property name="text">
           <string>Notifications Acknowledged:</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="notificationsAcknowledgedValue">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="bottomStatSpacer_MainScreen">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gameOverScreen" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>921</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(117, 238, 255)</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="gameOverVerticalLayout">
      <item>
       <spacer name="upperGameOverSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="gameOver">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>C059</family>
          <pointsize>32</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Game Over!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="underGameOverSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="distractedDrivingStatLayout">
          <item>
           <widget class="QLabel" name="drivingStats1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>NATS</family>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;According to the Insurance Information Institute there were 3,211 fatal crashes in 2021, that were caused by distracted driving.&lt;br/&gt;&lt;br/&gt;According to NHTSA, people aged 16-24 are most likely to use their cellphones while driving.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="drivingStats2">
            <property name="font">
             <font>
              <family>NATS</family>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Using NHTSA data, Forbes calculated that in Utah, 6.78% of fatal car accidents involved a distracted driver. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="statsHorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="gameStats">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>NATS</family>
            <pointsize>15</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="lowerGameOverSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="gameOverStatsLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <spacer name="gifLeftSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="gameOverGif">
          <property name="text">
           <string>GIF</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="gifRightSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>startUpScreen</zorder>
   <zorder>gameOverScreen</zorder>
   <zorder>mainScreen</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>World</class>
   <extends>QWidget</extends>
   <header location="global">world.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
